examples/lock-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -59,7 +60,7 @@ void AppDeviceCallbacks::PostAttributeChangeCallback(EndpointId endpointId, Clus
 void AppDeviceCallbacks::OnOnOffPostAttributeChangeCallback(EndpointId endpointId, AttributeId attributeId, uint8_t * value)
 {
     VerifyOrExit(attributeId == app::Clusters::OnOff::Attributes::OnOff::Id,
-                 ESP_LOGI(TAG, "Unhandled Attribute ID: '0x%04x", attributeId));
+                 ESP_LOGI(TAG, "Unhandled Attribute ID: '0x%04" PRIx32, attributeId));
```

Comment:
Why 04?  I know it was there, but it makes no sense.

Reply:
Yes, this doesn't make sense, fixed this as well.